modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -34,3 +34,25 @@ $ oc describe csr <csr_name> <1>
 ----
 $ oc adm certificate approve <csr_name>
 ----
+
+In case your load balancer certs are also expired you will need to talk to one of the masters directly.
+
+. Setup an ssh tunnel
++
+[source,terminal]
+----
+$ ssh -i <master_private_key> -N -L 42443:localhost:6443 -o ProxyCommand='ssh -i <bastion_private_key> -W %h:%p <bastion_user>@<bastion>' core@<master>
+----
+<1> `<master_private_key>` is the ssh key to to use to connect to your master.
+<2> `<bastion_private_key>` is the ssh key to to use to connect to your bastion.
+<3> `<bastion>` is the IP or DNS name of your bastion.
+<4> `<master>` is the IP or DNS name of your master node.
+
+. Adjust all of your `oc` calls to use this tunnel like
++
+[source,terminal]
+----
+$ oc --server https://localhost:42443 get csr
+----
+
+Note that your current kubeconfig context has to use certificate based authentication. Using token based authentication will not work as it requires additional components to be running.
```

Comment:
Just so that I understand - using token-based auth will not work for this entire procedure of recovering from the expired control plane certs? They have to use certificate-based auth for it to be successful (this is not just specific to the load balancer certs also being expired)?